src/moin/items/__init__.py

```diff
@@ -1568,8 +1570,8 @@ def do_modify(self, item_may=None):
                 item = Item.create(template_name)
                 form = self.ModifyForm.from_item(item)
                 # replace template name with new item name and remove TEMPLATE tag
-                form["meta_form"]["name"] = self.name
-                form["meta_form"]["tags"].remove(TEMPLATE)
+                form["meta_form"][NAME] = self.name
+                form["meta_form"][TAGS].remove(TEMPLATE)
```

Comment:
These are keys into a **form** dict, not into a revision meta dict.

So, while it is the same value, it's a different use case.

Reply:
Agree. Reverted.